src/interface_r/R/install.R

```diff
@@ -0,0 +1,14 @@
+#' Install h2o4gpu from a path to wheel file
+#' @export
+install_h2o4gpu <- function(wheel_path) {
+  # TODO: Support other install methods (future work)
+  conda_env <- "r-h2o4gpu"
+  if (!conda_env %in% reticulate::conda_list()$name) {
+    conda_create(envname = conda_env)
+  } else {
+    conda_remove(envname = conda_env)
+    conda_create(envname = conda_env)
+  }
+  use_condaenv(conda_env)
+  conda_install(envname = conda_env, packages = wheel_path, pip = TRUE, pip_ignore_installed = TRUE)
+}
```

Comment:
So the idea here is to create a conda_env `r-h2o4gpu` no matter what?

Reply:
Or a virtualenv (future work) named `r-h2o4gpu` (unfortunately this has to be a hard-coded string). In TensorFlow R binding, we had something similar `r-tensorflow`. This way R users don't need to activate the environments themselves. We can provide this as R `options` too so if any users want to change this, they can do so as well.

Reply:
e.g. 
```
options(h2o4gpu_env_name='r-h2o4gpu-2')
options(h2o4gpu_env_load_method='virtualenv')
```

Reply:
Gotcha and why is the `virtualenv` future work? Is it not available within reticulate?

Reply:
It's not fully available yet. There are `use_conda`, `conda_create`, `conda_list`, etc. but there's only `use_virtualenv` at this moment.

Reply:
Ah, I see. I think we can expose that for folks who know how to build a virtualenv. What do you think of that?

Reply:
You mean exposing those options?